config.py

```diff
@@ -3,3 +3,4 @@
 
 class Config:
     SECRET_KEY = os.environ.get('SECRET_KEY')
+    NASA_API = 'https://api.nasa.gov{path}'
```

Comment:
use `endpoint` word here, and add `/` after domain -> `https://api.nasa.gov/{endpoint}`